Library/Homebrew/exceptions.rb

```diff
@@ -478,17 +488,20 @@ def initialize(formula, cmd, args, env)
     super "Failed executing: #{cmd} #{pretty_args}".strip
   end
 
+  sig { returns(T::Array[T.untyped]) }
```

Comment:
No way we can type these?

Reply:
From the code I couldn't guess what the type would be beyond an array of things